app/src/main/java/org/mifos/selfserviceapp/ui/fragments/FundTransferFragment.java

```diff
@@ -0,0 +1,209 @@
+import android.widget.Spinner;
+import android.widget.TextView;
+import android.widget.Toast;
+
+import org.mifos.selfserviceapp.R;
+import org.mifos.selfserviceapp.data.FundTransfer.FundTransferRequest;
+import org.mifos.selfserviceapp.data.FundTransfer.FundTransferResponse;
+import org.mifos.selfserviceapp.data.FundTransfer.FundTransferTemplate;
+import org.mifos.selfserviceapp.data.FundTransfer.FundTransferTemplateResponse;
+import org.mifos.selfserviceapp.presenters.FundTransferPresenter;
+import org.mifos.selfserviceapp.ui.activities.BaseActivity;
+import org.mifos.selfserviceapp.ui.views.FundTransferView;
+
+import java.util.List;
+
+import javax.inject.Inject;
+
+import butterknife.BindView;
+import butterknife.ButterKnife;
+import butterknife.OnClick;
+
+/**
+ * @author Vishwajeet
+ * @since 22/8/16.
+ */
+
+public class FundTransferFragment extends Fragment implements FundTransferView {
+
+    @Inject
+    FundTransferPresenter mFundTransferPresenter;
+
+    private View rootView;
+    private long fromClientId;
+    private long fromOfficeId;
+    private long fromAccountId;
+    private long fromAccountType;
+    private long toClientId;
+    private long toOfficeId;
+    private long toAccountId;
+    private long toAccountType;
+
+    @BindView(R.id.tv_transfer_date)
+    TextView tvTransactionDate;
+    @BindView(R.id.et_transfer_date)
+    EditText etTransferDate;
+    @BindView(R.id.tv_transfer_description)
+    TextView tvTransferDescription;
+    @BindView(R.id.et_transfer_description)
+    EditText etTransferDescription;
+    @BindView(R.id.tv_amount_transfer)
+    TextView tvAmountTransfer;
+    @BindView(R.id.et_amount_transfer)
+    EditText etAmountTransfer;
+    @BindView(R.id.tv_from_account)
+    TextView tvFromAccount;
+    @BindView(R.id.sp_from_account)
+    Spinner spFromAccount;
+    @BindView(R.id.tv_to_account)
+    TextView tvToAccount;
+    @BindView(R.id.sp_to_account)
+    Spinner spToAccount;
+    @BindView(R.id.bt_transfer_amount)
+    Button btTransferAmount;
+    @BindView(R.id.bt_cancel_payment)
+    Button btCancelPayment;
+
+    @Override
+    public void onCreate(@Nullable Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        ((BaseActivity) getActivity()).getActivityComponent().inject(this);
+    }
+
+    @Nullable
+    @Override
+    public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
+        rootView = inflater.inflate(R.layout.fragment_fund_transfer, container, false);
+        ButterKnife.bind(this, rootView);
+
+        mFundTransferPresenter.attachView(this);
+
+        mFundTransferPresenter.loadAccountTransferTemplate();
+
+        return rootView;
+    }
+
+    @Override
+    public void showFundTransferTemplate(final FundTransferTemplateResponse fundTransferTemplate) {
+        List<FundTransferTemplate> fromAccountNumber = fundTransferTemplate.getFromAccountOptions();
```

Comment:
It's useful to call the variable fromAccountNumberList to make it clear.. otherwise it looks like it's an account number.